lib/smoke_detector/providers/rollbar.rb

```diff
@@ -1,4 +1,6 @@
 require 'json'
+require 'rollbar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.